test/JHipsterNetSampleApplication.Test/Web/Rest/BankAccountResourceIntTest.cs

```diff
@@ -202,7 +227,7 @@ public async Task UpdateBankAccount()
             var updatedBankAccount =
                 await _applicationDatabaseContext.BankAccounts.SingleOrDefaultAsync(it => it.Id == _bankAccount.Id);
             // Disconnect from session so that the updates on updatedBankAccount are not directly saved in db
-//TODO detach
+            //TODO detach
```

Comment:
Please stay consistent with the formatting to prevent meaningless lines to review.